specRunner.js

```diff
@@ -0,0 +1,36 @@
+console.log("Running Tests ... \n");
+
+// require('./tests/airportFunctions.spec')
+
```

Comment:
GOOD 1: This spec runner file is very clean, well formatted, and clearly commented.
BAD 1: A small thing but the comment of an old test still exists at thop
GOAL: Review code before pusing it as to not leave old code or bad comments